utils/api/benefitHandler.ts

```diff
@@ -383,7 +383,7 @@ export class BenefitHandler {
     // Check OAS. Does both Eligibility and Entitlement, as there are no dependencies.
     // Calculate OAS with and without deferral so we can compare totals and present more beneficial result
 
-    if (this.input.client.receiveOAS) {
+    if (this.input.client.receiveOAS && !this.input.client.livedOnlyInCanada) {
```

Comment:
wasn't there a case for which we removed the condition?

Reply:
Talked to Vero - we will accept this change and trust the automated tests to expose any potential bugs that we brought back by making this change.